common/cpp/react/renderer/components/RNCViewPager/RNCViewPagerShadowNode.cpp

```diff
@@ -0,0 +1,35 @@
+#include "RNCViewPagerShadowNode.h"
```

Comment:
```suggestion
// https://github.com/facebook/react-native/blob/main/ReactCommon/react/renderer/components/scrollview/ScrollViewShadowNode.cpp
#include "RNCViewPagerShadowNode.h"
```

Reply:
any way to generate that with codegen or suggest the feature request?

Reply:
Right now, the codegen generates those files inside a build folder. We can suggest a feature request and request a comment from other developers, on what they think about it.